src/components/Modal/CustomModal.jsx

```diff
@@ -0,0 +1,24 @@
+import Portal from '../Portal/Portal';
+import ModalBox from './ModalComponents/ModalBox';
+import TestModalComponent from './ModalContent/TestModalContent'; //예시 1.모달의 마크업 컴포넌트를 import해서 content에 전달해줍니다.
+/**
+ * CustomModal props list
+ *@property {Hook} isModalOpen               - 부모의 정의되어있는 isModalOpen // useState(블리언값) 값을 전달받습니다. (부모와 name이 동일해야합니다.)
+ *@property {function} setIsModalOpen        - 부모의 정의되어있는 setIsModalOpen //useState(set함수)를 전달받습니다.(부모와 name이 동일해야합니다.)
+ *@property {function} handleModalToggle     - 부모의 정의되어있는 handleModalToggle 토글함수를 전달받습니다. (부모와 name이 동일해야합니다.)
+ */
+
+const CustomModal = (isModalOpen, setIsModalOpen, handleModalToggle) => {
```

Comment:
TestModal과 구조가 같아 보이는데, CustomModal에서는 props에 {}중괄호가 빠져 있네요?
코드 실행해보니 중괄호가 있는게 맞는 것 같네요. 
확인 부탁드립니다~

```suggestion
const CustomModal = ({isModalOpen, setIsModalOpen, handleModalToggle}) => {
```